.github/workflows/validate-install-from-source.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         - image: tgagor/centos-stream
         - image: redhat/ubi8
         - image: alpine
+        - image: alpine:3.14.10
```

Comment:
Not sure how I feel about this - I'd prefer to continue using the latest version in this workflow but could be persuaded otherwise.

Reply:
I share that sentiment. If there is a reasonable way to keep targeting the latest version (which many of our users will do, I expect), I'd love to see that.

Reply:
We are targeting both!

```
- image: alpine
- image: alpine:3.14.10
```

The first line is the latest, the second is a specific tag/version.

Reply:
Sorry I missed that! Was reviewing from my phone 😂.